core/src/main/java/com/exadel/etoolbox/backpack/core/services/pckg/impl/DeletePackageServiceImpl.java

```diff
@@ -0,0 +1,62 @@
+package com.exadel.etoolbox.backpack.core.services.pckg.impl;
+
+import com.exadel.etoolbox.backpack.core.services.pckg.BasePackageService;
+import com.exadel.etoolbox.backpack.core.services.pckg.DeletePackageService;
+import org.apache.jackrabbit.vault.packaging.JcrPackage;
+import org.apache.jackrabbit.vault.packaging.JcrPackageManager;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+
+/**
+ * Implements {@link DeletePackageService} to provide delete package operations
+ */
+@Component(service = DeletePackageService.class)
+public class DeletePackageServiceImpl implements DeletePackageService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DeletePackageServiceImpl.class);
+
+    @Reference
+    private BasePackageService basePackageService;
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void delete(ResourceResolver resourceResolver, String path) {
```

Comment:
I think this method must have a return value depending on the actual result of the operation (see also my very last comment down below)